fidesctl/tests/core/test_dataset.py

```diff
@@ -382,20 +384,20 @@ def database_setup(self):
     def test_get_db_tables(self, request, database_type):
         print(request.node.get_closest_marker("external"))
         print(request.keywords)
-        database_parameters = TEST_DATABASE_PARAMETERS.get(database_type)
```

Comment:
`.get` shouldn't be used unless its actually ok for something to be `None`, otherwise it makes it harder to debug things down the stack. This should throw an error immediately if the `database_type` doesn't exist

Reply:
not _universally_, as it is super handy in some situations. The following examples are places where `.get` makes things simpler.

```python
try:
    foo = some_dict["foo"]
except KeyError:
    foo = None
```

or

```python
foo = some_dict["foo"] if "foo" in some_dict.keys() else None
```

but in these test cases, they _should_ actually fail if there is a `KeyError`, so they serve to obfuscate the error

Reply:
Thank you for explaining that ! 🙌🏽